src/csi/pkg/pos/controllerserver.go

```diff
@@ -48,17 +49,25 @@ import (
 const (
 	CREATING string = "creating"
 	CREATED         = "created"
+	MAXIMUM_VOLUMES = 4
```

Comment:
Why the maximum volumes are set to 4 per target. Any specific reason for that?

Reply:
Sorry I am late to reply. Actually, its for our OSS service. We are supposed to use 4 poseidon servers and create 16 volumes for minio service.  So, i set maximum volumes, POS can have, as 4 to distribute 16 volumes to 4 poseidon servers evenly.

Reply:
Thanks for the clarification. In that case, I think it will be better to move the maximum volumes also to storageclass.yaml. So, that it is configurable. This is because other systems which also wants to use the csi driver will not be restricted by the count 4.